### PR TITLE
Cleaner and more verbose failure

### PR DESCRIPTION
**Pull request by adammhaile**
Brew URLs now printed to console
Hard fail on any brew build fail.
Failure logs printed to console if brew fails

### CODE REVIEW

**Comment by adammhaile** [comment]
@jupierce

File: build-scripts/ose_images/ose_images.sh [hunk 1]
```diff
@@ -426,31 +428,32 @@ check_builds() {
 
     if [ "$state" == "open" -o "$state" == "free" ]; then # free state means brew is waiting for a free builder
         echo "brew build for $package is still running..."
-    elif [ "$state" == "closed" ]; then
-        echo "==== ${package} IMAGE COMPLETED ===="
```
Comment (stevekuznetsov) [inline build-scripts/ose_images/ose_images.sh]: misleading diff due to whitespace?

Comment (adammhaile) [inline build-scripts/ose_images/ose_images.sh]: Yeah... even I second guessed myself when I saw the diff. I took the elif/else parts and indented one level so they were their own if/else with a single call to `mv` at the end. I can confirm it's not just a whitespace change.

Comment (adammhaile) [inline build-scripts/ose_images/ose_images.sh]: It's much more clear in the side by side split diff view.

**Pull request merged**